### DISCUSSION TITLE
Update error

### DISCUSSION THREAD

**Discussion by JustMeWho** [rejetto/hfs#172] [General]
Hi,
I tried clicking on the update option but it doesn't work. After clicking the update button, it throws an error and then the server shuts down. Maybe this could be fixed in a future update...

![image](https://user-images.githubusercontent.com/62073281/226952082-eb19a543-7ced-4a63-9986-4062192e014f.png)

**Comment by rejetto** [comment]
i tried and it worked here.
Please check what error was printed on the HFS console window (if any), otherwise I have no clue.
